.github/workflows/continuous-delivery.yaml

```diff
@@ -43,8 +43,8 @@ jobs:
         name: Authenticate
         uses: actions/checkout@v2
         with:
-          fetch-depth: 1
           persist-credentials: true
+          ref: master
```

Comment:
I have trouble understanding why is it necessary. Did you test it on some other repositories?